Aquarius/Src/Aquarius/Core/Window.cpp

```diff
@@ -0,0 +1,109 @@
+#include "Window.h"
+#include "Log.h"
+#include <assert.h>
+
+namespace Aquarius {
+
+    Window::Window(uint32_t width, uint32_t height, std::string &&name, bool vsync)
+        : m_Width(width), m_Height(height), m_Name(std::move(name)), m_VsyncEnabled(vsync), m_Window(nullptr), m_Context(nullptr)
```

Comment:
What are we going with for max line length? Not that it matters but could put each of these on their own line

Reply:
I was thinking about that yea, I dont think we should worry too much about 80 chars but just keep it reasonable hahaha

Reply:
Agreed, and with so many arguments I think this may look stupid when they are all on their own line so it can probably stay as-is